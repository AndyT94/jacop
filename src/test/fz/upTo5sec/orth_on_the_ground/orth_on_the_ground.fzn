array [1..6] of var int: orthotope:: output_array([1..2,1..3]) = [1,2,3,2,3,5];
solve  satisfy;
