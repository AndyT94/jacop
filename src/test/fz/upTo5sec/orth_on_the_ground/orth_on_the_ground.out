orthotope = array2d(1..2, 1..3, [1, 2, 3, 2, 3, 5]);
----------